File: 1206-corporate-flight-bookings/1206-corporate-flight-bookings.cs
public class Solution {
    public int[] CorpFlightBookings(int[][] bookings, int n) {
        
        int acc = 0; 
        int[]x= new int[n+2];
        for(int i = 0; i< bookings.Length; i++)
        {
            int s = bookings[i][0];
            int e = bookings[i][1];
            int val = bookings[i][2];
            x[s]+= val;
            x[e+1]-=val;
        }int[]res = new int[n];
        for(int i= 0; i<n; i++)
        {
            acc += x[i+1];
            res[i] = acc;
        }return res; 
    }
}

File: 1206-corporate-flight-bookings/1206-corporate-flight-bookings.js
/**
 * @param {number[][]} bookings
 * @param {number} n
 * @return {number[]}
 */
var corpFlightBookings = function(bookings, n) {

    let acc = 0; 
        let x= new Array(n+2).fill(0);
        for(let i = 0; i< bookings.length; i++)
        {
            let s = bookings[i][0];
            let e = bookings[i][1];
            let val = bookings[i][2];
            x[s]+= val;
            x[e+1]-=val;
        }let res = new Array(n).fill(0);
        for(let i= 0; i<n; i++)
        {
            acc += x[i+1];
            res[i] = acc;
        }return res; 
    
};